packages/types/formbricks-surveys.ts

```
@@ -24,6 +24,9 @@ export interface SurveyBaseProps {
   languageCode: string;
   dir?: "ltr" | "rtl" | "auto";
   setDir?: (dir: "ltr" | "rtl" | "auto") => void;
+  /** Notifies the host of the survey's active language code (e.g. "default", "en-AU", "he").
+   *  Link surveys use it to keep <html lang>/<dir> in sync; embedded widgets omit it. */
```

ESLint

[error] tsdoc-html-tag-missing-equals: The HTML element has an invalid attribute: Expecting "=" after HTML attribute name

(tsdoc/syntax)